applications_tests/gls_nitsche_navier_stokes_33/noslip33_simplex.prm

```diff
@@ -0,0 +1,148 @@
+# Listing of Parameters
+# ---------------------
+
+
+# --------------------------------------------------
+# Simulation and IO Control
+#---------------------------------------------------
+subsection simulation control
+  set method                  = steady
+  set output name             = noslip33_simplex
+  set log precision           = 0
+end
+
+#---------------------------------------------------
+# FEM
+#---------------------------------------------------
+subsection FEM
+    set velocity order        = 1
+    set pressure order        = 1
+end
+
+# --------------------------------------------------
+# Mesh Adaptation Control
+#---------------------------------------------------
+subsection mesh adaptation
+  set type                    = uniform
+end
+
+#---------------------------------------------------
+# Timer
+#---------------------------------------------------
+subsection timer
+    set type    = none
+end
+
+#---------------------------------------------------
+# Force
+#---------------------------------------------------
+subsection forces
+    set verbosity             = verbose   # Output force and torques in log <quiet|verbose>
+    set calculate forces      = true      # Enable force calculation
+    set calculate torques     = false     # Enable torque calculation
+    set force name            = force     # Name prefix of force files
+    set torque name           = torque    # Name prefix of torque files
+    set output precision      = 10        # Output precision
+    set calculation frequency = 1         # Frequency of the force calculation
+    set output frequency      = 1         # Frequency of file update
+end
+
+#---------------------------------------------------
+# Physical Properties
+#---------------------------------------------------
+subsection physical properties
+    set kinematic viscosity            = 1.0
+end
+
+#---------------------------------------------------
+# Mesh
+#---------------------------------------------------
+subsection mesh
+	set type = dealii
+	set grid type = subdivided_cylinder
+	set grid arguments = 10 : 0.75 : 0.7
+	set initial refinement = 1
+	set simplex = false
+end
+
+#---------------------------------------------------
+# Nitsche
+#---------------------------------------------------
+subsection nitsche
+	set beta = 10
+  	set calculate torques on solid = true
+  	set verbosity = verbose
+  	set number of solids = 1
+
+  	subsection nitsche solid 0
+	  	subsection mesh
+	  		set simplex = true
+		  	set type = gmsh
+			set file name = ../../sphere.msh
```

Comment:
could deal.ii mesh generator works for simplex as of now? If yes, I think it would be better to use it rather than an external .msh. If no, perhaps leave a note (on the git issues tab?) so it can be changed when it does.

Reply:
That's a good remark, because in theory you could use deal.II meshes with simplices. Maybe not with this test case, but eventually yeah. This would be a good "TODO".

Reply:
It didn't work when I coded it, but maybe it's because I had an old version of dealii, so I'll make sure, and if not possible for now I'll make a note of it for later (TODO or issues)